File: Web/RMA-Web-Shell/projects/claimcare/src/app/claim-manager/shared/entities/stm-overview.ts

export class StmOverview {
  eventTypeId: number;
  personEventId: number;
  claimTypeId: number;
  insuredLifeId: number;
  personEventBucketClassId: number;
  insuranceTypeId: number;
  suspiciousTransactionStatusId: number;
  suspiciousTransactionName: string;
  isStraightThroughProcess: boolean;
  name: string;
  claimId: number;
  policyId?: number;
  createdDate: Date;
  filter: boolean;
  startDate: Date;
  endDate: Date;
}

File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/StmOverview.cs

using System;

namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class StmOverview
    {
        public int EventTypeId { get; set; }
        public int PersonEventId { get; set; }
        public int ClaimTypeId { get; set; }
        public int InsuredLifeId { get; set; }
        public int PersonEventBucketClassId { get; set; }
        public int InsuranceTypeId { get; set; }
        public int SuspiciousTransactionStatusId { get; set; }
        public string SuspiciousTransactionName { get; set; }
        public bool IsStraightThroughProcess { get; set; }
        public string Name { get; set; }
        public int ClaimId { get; set; }
        public int? PolicyId { get; set; }
        public DateTime CreatedDate { get; set; }
        public bool Filter { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
    }
}